norduser/service/fork.go

```diff
@@ -191,7 +191,7 @@ func (c *ChildProcessNorduser) StopAll() {
 
 	for _, pid := range pids {
 		if err := syscall.Kill(pid, syscall.SIGTERM); err != nil {
-			log.Println("failed to send a signal to norduserd: ", err)
+			log.Println(internal.ErrorPrefix, "failed to send a signal to norduserd: ", err)
```

Comment:
There are many more log calls in those file without prefixes or with inconsistent formatting. So I think that we should fix all of them or do not touch them at all in this PR. Also, when `log.Println` is used, spaces between arguments are added automatically, so space after colon in `"Message: "` results in double spaces.

Reply:
Yes, there are many, because of this I don't think we should fix them in this PR.